README.md

```diff
@@ -1,3 +1,69 @@
-# startable-standard
+# The StarTable data format
+
+StarTable is a human- and machine-readable data format designed to conveniently
+store an arbitrary number of two-dimensional tables of data. It also supports peripheral information elements such as metadata and application-specific functionality. 
+
+The StarTable format is file-format agnostic. In practice, CSV files and Excel workbooks are most commonly used. But any file format that can be used to represent a set of
+tables, each with columns and rows of cells, can, in principle, adhere to the StarTable format.
+
+## Parsers and utilities
+
+StarTable parsers exist for Python, MATLAB, and C#. They can currently parse StarTable files in CSV and Excel format. 
```

Comment:
There is also a C++ parser, written by Søren if I'm not wrong. Should we also open-source that one?

Reply:
@fuglede ^^^ ?

Reply:
That's right. In terms of specification adherence that one *only* implements what was necessary for JackTop. As such I don't think it's particularly relevant here.

See `jacktop/ROSAbucint/trunk/src/startable.cpp` in the internal SVN repos if you're interested.